web-extension/support/isekaiscan.js

```diff
@@ -7,10 +7,18 @@ function IsekaiScanComPlugin() {
 
 IsekaiScanComPlugin.prototype.getInfos = function(url, doc) {
     let parts = url.split('/').filter(s => s.length !== 0);
+    let chapter_prefix = null;
+
+    if (url.indexOf('chapter-') !== -1) {
+        chapter_prefix = 'chapter';
+    }
+    else if (url.indexOf('ch-') !== -1) {
```

Comment:
In new "code age", you can put `else if` on the same line as the closing paren. :p

Reply:
if we want to stick to that, we should really include that in the linter.